toolchain/ltrace.py

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env python
```

Comment:
Hello @jayarama-ibm would you please add the license heading? You can inspire here: https://github.com/avocado-framework/avocado-misc-tests/pull/26